ivy/functional/ivy/norms.py

```diff
@@ -28,8 +37,46 @@ def layer_norm(x, normalized_idxs, epsilon=None, scale=None, offset=None, new_st
 
     Returns
     -------
-    ret
+     ret
         The layer after applying layer normalization.
+    
+    Example
+    -------
+
```

Comment:
You need to put headers like following as mentioned in the contributor guidelines (https://lets-unify.ai/ivy/deep_dive/11_docstring_examples.html):

```
Functional Examples
-------------------

With :code:`ivy.Array` input:
```